github/Commit.pyi

```diff
@@ -41,8 +42,13 @@ class Commit(CompletableGithubObject):
     ) -> CommitStatus: ...
     @property
     def files(self) -> List[File]: ...
+    def get_check_suites(
+        self,
+        app_id: Union[_NotSetType, int],
+        check_name: Union[_NotSetType, str],
+    ) -> PaginatedList[CheckSuite]: ...
     def get_combined_status(self) -> CommitCombinedStatus: ...
-    def get_comments(self) -> PaginatedList[GitCommit]: ...
+    def get_comments(self) -> PaginatedList[CommitComment]: ...
```

Comment:
Wasn't that already merged?

Reply:
No, I rebased #1646 on the latest upstream which does not include that change: https://github.com/PyGithub/PyGithub/blob/master/github/Commit.pyi#L45

Reply:
Okay, let me break that out into a quick PR.

Reply:
And meged

Reply:
Done ✅